tests/app/main/views/test_suppliers.py

```diff
@@ -1068,6 +1091,9 @@ def test_should_show_agreement_details_on_page(self, s3, data_api_client):
         assert len(document.xpath('//li[contains(text(), "DG9 0QG")]')) == 1
         # Company number - linked
         assert len(document.xpath('//a[@href="https://beta.companieshouse.gov.uk/company/1234456"][contains(text(), "1234456")]')) == 1  # noqa
+        # Lots
+        assert len(document.xpath('//li[contains(text(), "Lettuce & cucumber")]')) == 1
```

Comment:
Yeah I was thinking about this - my first instinct was to try and find a way of getting lxml to tell me what character an element was found at but no dice... I'll think...